frontend/src/types/global.d.ts

```diff
@@ -7,9 +7,6 @@ enum UserType {
     None
 }
```

Comment:
Mmm got this question only now, but still. Why to use `.d.ts` here? Why not just pure `.ts`?

`.d.ts` files are needed when we want to combine typescript code with javascript code, which doesn't seem to be the case here

Reply:
hmm, I thought that .d.ts files are using for keeping global types, functions and constants
but may be I'd read some fakes...